crates/edr_provider/src/requests.rs

```diff
@@ -14,18 +14,7 @@ pub use crate::requests::methods::{MethodInvocation, OneUsizeOrTwo, U64OrUsize};
 #[serde(untagged)]
 pub enum ProviderRequest {
     /// A single JSON-RPC request
-    Single(Request),
+    Single(MethodInvocation),
```

Comment:
I thought this had been renamed to `..Request`?

Reply:
I thought `Request` in itself might be a bit confusing, because it doesn't contain all the fields of a JSON-RPC request. Happy to rename it though

Reply:
Ah ok. Yeah for the RPC client I named it `RequestMethod`, but for the RPC "server" `MethodInvocation` makes more sense to me. Happy to change it to `RequestMethod` though